app/src/test/java/util/PDFUtilsTest.java

```diff
@@ -0,0 +1,82 @@
+package util;
+
+import org.junit.Test;
+
+import swati4star.createpdf.util.PDFUtils;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class PDFUtilsTest {
+    private static final int NUM_OF_PAGES = 23;
+    private static final int NO_ERROR = 0;
+    private static final int ERROR_PAGE_NUMBER = 1;
+    private static final int ERROR_RANGE = 2;
+    private static final int ERROR_INVALID_INPUT = 3;
+
+    @Test
+    public void checkRange_CorrectRanges_Returns_NO_ERROR() {
```

Comment:
Don't use snake case please. It's Java and not Python.

Reply:
But in java constants are named like this.. that's why I did so.
AndI named function like this because in fileUtilsTest.java file, it is done like this.

Reply:
> But in java constants are named like this.

Yes, but they are all caps and this is not a constant.

> AndI named function like this because in fileUtilsTest.java file, it is done like this.

Then that is wrong. You can create an issue and fix it.

Reply:
Just to confirm.
I only need to make change in test **name** and make it camelCase?

Reply:
yes. just make these camel case

Reply:
done

Reply:
@Swati4star should I make a new issue for wrong naming in fileUtilsTest.java or you will do it?

Reply:
leave that. I'll change those later